src/thriveopedia/ThriveopediaManager.cs

```diff
@@ -0,0 +1,32 @@
+﻿using System;
+using Godot;
+
+/// <summary>
+///   Utility for universally accessible Thriveopedia actions, such as opening a page by its name.
+/// </summary>
+public class ThriveopediaManager
+{
+    private static readonly ThriveopediaManager ManagerInstance = new();
+
+    public static ThriveopediaManager Instance => ManagerInstance;
+
+    /// <summary>
+    ///   Action to open the Thriveopedia in the current game context, e.g. from the main menu or from the pause menu.
+    /// </summary>
+    public Action<string>? OpenCallback { get; set; }
```

Comment:
This not being an event (that can have multiple listeners registered at once) is probably fine, it wouldn't make sense anyway for multiple things to react to this and try to open their own thriveopedias at once.

Reply:
It could be an event, but functionally there should only ever be one action attached, so that would come with the added complexity of removing listeners every time and the risk of bugs associated with that.

Reply:
Updated.